pokemon-showdown

```diff
@@ -16,15 +18,18 @@ try {
 try {
 	require.resolve('sockjs');
 } catch (e) {
-	let command = 'npm install --production';
-	console.log('Installing dependencies: `' + command + '`...');
-	require('child_process').spawnSync('sh', ['-c', command], {stdio: 'inherit'});
+	console.log('Installing dependencies...');
+	spawnSync('npm', ['install', '--production'], {stdio: 'inherit', shell: true});
```

Comment:
I don't, actually. I was mixing up `.exe` files with `.bat`/`.cmd` ones when it comes to which of them need the shell.